### PR TITLE
Fixes for MERCHANTS-14

### PR DESCRIPTION
**Pull request by guilhermegeru**
Fix [suggestions](https://github.com/geru-br/keyloop/pull/20#pullrequestreview-375203090) of [MERCHANTS-14](https://geruteam.atlassian.net/browse/MERCHANTS-14) review, and additional improvements from [MERCHANTS-65](https://geruteam.atlassian.net/browse/MERCHANTS-65)

**Changes:**
- Modify `get_by` to `get` in Identity
- Remove `PermissionGrant` interfaces
- Move validation of `identity_id` from view to specific path schema (`IdentityPathSchema`)
- Add logs

### CODE REVIEW

File: keyloop/api/v1/permission_grant.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def __acl__(self):
 
 class CollectionPostSchema(marshmallow.Schema):
     path = marshmallow.fields.Nested(BasePathSchema)
-    body = marshmallow.fields.Nested(PermissionGrantSchema)
+    body = marshmallow.fields.Nested(PermissionGrantSchema())
```
Comment (doubleG-geru) [inline keyloop/api/v1/permission_grant.py]: Pq esses parenteses? Deixar a surgestão aqui:
```suggestion
    body = marshmallow.fields.Nested(PermissionGrantSchema)
```

**Review by doubleG-geru — APPROVED** [review]
Apenas um comentário, mas LGTM

File: keyloop/schemas/path.py [hunk 2]
```diff
@@ -36,3 +35,23 @@ def validate_realm(self, value):
...
+        except IdentityNotFound:
+            request.errors.add(
+                location='path',
+                name='identity_id',
+                description='Identity not found'
+            )
+            request.errors.status = 404
+            return
+
+        request.identity = identity
```
Comment (jsbueno-geru) [inline keyloop/schemas/path.py:57]: LGTM

**Review by jsbueno-geru — APPROVED** [review]
:+1:

**Pull request merged**